src/public/modules/forms/base/directives/field.client.directive.js

```diff
@@ -41,6 +41,9 @@ function fieldDirective(FormFields, $location, $sanitize) {
         query.length > 1 ? querystring.parse(query[1]) : undefined
 
       if (
+        !scope.field.myInfo && // disallow prefill for myinfo
+        scope.field.allowPrefill &&
+        scope.field.allowPrefill === true &&
```

Comment:
redundant, `scope.field.allowPrefill` is enough since it will go to the next conditional only if it's truthy

Reply:
agree